renovate.json

```diff
@@ -1,47 +1,5 @@
 {
 "extends": [
-    "config:base"
-  ],
-  "dependencyDashboard": true,
-  "logFileLevel": "trace",
-  "enabledManagers": ["gomod"],
-  "postUpdateOptions": ["gomodTidy"],
-  "constraints": {
-    "go": "1.19"
-  },
-  "schedule":[
-    "every weekend"
-  ],
-  "packageRules": [
-    {
-      "matchPackageNames": ["github.com/openstack-k8s-operators/keystone-operator/api"],
-      "enabled": false
-    },
-    {
-      "groupName": "openstack-k8s-operators",
-      "matchPackagePrefixes": ["github.com/openstack-k8s-operators"],
-      "excludePackageNames": ["github.com/openstack-k8s-operators/keystone-operator/api"],
-      "schedule": [
-        "every weekend"
-      ]
-    },
-    {
-      "groupName": "k8s.io",
-      "matchPackagePatterns": ["^k8s.io"],
-      "excludePackagePatterns": ["^k8s.io/kube-openapi"],
-      "allowedVersions": "< 0.27.0",
-      "enabled": true
-    },
-    {
-      "groupName": "sigs.k8s.io/controller-runtime",
-      "matchPackagePatterns": ["^sigs.k8s.io/controller-runtime"],
-      "allowedVersions": "< 0.15.0",
-      "enabled": true
-    }
-  ],
-  "postUpgradeTasks": {
-    "commands": ["make gowork", "make tidy", "make manifests generate"],
```

Comment:
It seems the common config regresses this as it only runs make tidy https://github.com/openstack-k8s-operators/renovate-config/blob/79f5b0ba87339a43b8c7bb9f9391aa3f7152fab1/default.json#L47

Reply:
okay, so I guess we can drop postUpgrade from common and then just have it in each operator. Probably safer that way as it depends on the Makefile too right

Reply:
https://github.com/openstack-k8s-operators/renovate-config/pull/1

Reply:
+1 sounds good to me

Reply:
I'm OK with that as I can imagine some repository does not have manifests and generate targets.